lib/sproxyd.js

```diff
@@ -151,8 +151,15 @@ class SproxydClient {
         let counter = tries;
         log.info('request', { method, value, args, counter });
 
+        let getResponse = false;
         this._handleRequest(method, stream, size, key, log, (err, ret) => {
             if (err && !err.isExpected) {
+                if (getResponse === true) {
+                    const error = new Error('trying to put data after sproxyd' +
+                        ' send back a response, size of the object is may ' +
+                        'be wrong');
+                    throw error;
```

Comment:
note: I think for this kind of error, we should at least log once at the deepest level possible.